content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -588,6 +588,7 @@ parameters:
   it must reside in the same resource group as the cluster, and `location` is
   ignored. If a storage account is not provided, a new storage account will be
   created in the same resource group as the cluster.
+  * `resourceGroup`: Specify the resource group in which the Azure disk will be created. It must be an existing resource group name. If it is unspecified, the disk will be placed in the the same resource group as the current Kubernetes cluster.
```

Comment:
pls move this `resourceGroup` parameter to `New Azure Disk Storage Class (starting from v1.7.2)` section, thanks.

Reply:
also remove the first space in this line, thanks.